projects/Amlogic/devices/S912/options

```diff
@@ -0,0 +1,13 @@
+  # Kernel extra targets to build
+     KERNEL_UBOOT_EXTRA_TARGET="gxm_q200_2g.dtb gxm_q201_1g.dtb gxm_q201_2g.dtb"
+
+  # OpenGL-ES implementation to use (no / bcm2835-driver / gpu-viv-bin-mx6q / opengl-meson)
+     OPENGLES="opengl-meson-t82x"
```

Comment:
Just a nitpick with the formatting, is there extra whitespaces here?

Reply:
I will fix this when squashing, thanks.